packages/jsii-reflect/test/__snapshots__/jsii-tree.test.js.snap

```diff
@@ -74,6 +41,192 @@ exports[`jsii-tree --all 1`] = `
  │   │   └─┬ value property (experimental)
  │   │     ├── immutable
  │   │     └── type: number
+ │   ├─┬ class BinaryOperation (experimental)
+ │   │ ├── base: Operation
+ │   │ ├── interfaces: IFriendly
+ │   │ └─┬ members
+ │   │   ├─┬ <initializer>(lhs,rhs) initializer (experimental)
+ │   │   │ └─┬ parameters
+ │   │   │   ├─┬ lhs
+ │   │   │   │ └── type: @scope/jsii-calc-lib.Value
+ │   │   │   └─┬ rhs
+ │   │   │     └── type: @scope/jsii-calc-lib.Value
+ │   │   ├─┬ hello() method (experimental)
+ │   │   │ └── returns: string
+ │   │   ├─┬ lhs property (experimental)
+ │   │   │ ├── immutable
+ │   │   │ └── type: @scope/jsii-calc-lib.Value
+ │   │   └─┬ rhs property (experimental)
+ │   │     ├── immutable
+ │   │     └── type: @scope/jsii-calc-lib.Value
+ │   ├─┬ class Calculator (experimental)
+ │   │ ├── base: CompositeOperation
+ │   │ └─┬ members
+ │   │   ├─┬ <initializer>(props) initializer (experimental)
+ │   │   │ └─┬ parameters
+ │   │   │   └─┬ props
+ │   │   │     └── type: Optional<jsii-calc.CalculatorProps>
+ │   │   ├─┬ add(value) method (experimental)
+ │   │   │ ├─┬ parameters
+ │   │   │ │ └─┬ value
+ │   │   │ │   └── type: number
+ │   │   │ └── returns: void
+ │   │   ├─┬ mul(value) method (experimental)
+ │   │   │ ├─┬ parameters
+ │   │   │ │ └─┬ value
+ │   │   │ │   └── type: number
+ │   │   │ └── returns: void
+ │   │   ├─┬ neg() method (experimental)
+ │   │   │ └── returns: void
+ │   │   ├─┬ pow(value) method (experimental)
+ │   │   │ ├─┬ parameters
+ │   │   │ │ └─┬ value
+ │   │   │ │   └── type: number
+ │   │   │ └── returns: void
+ │   │   ├─┬ readUnionValue() method (experimental)
+ │   │   │ └── returns: number
+ │   │   ├─┬ expression property (experimental)
+ │   │   │ ├── immutable
+ │   │   │ └── type: @scope/jsii-calc-lib.Value
+ │   │   ├─┬ operationsLog property (experimental)
+ │   │   │ ├── immutable
+ │   │   │ └── type: Array<@scope/jsii-calc-lib.Value>
+ │   │   ├─┬ operationsMap property (experimental)
+ │   │   │ ├── immutable
+ │   │   │ └── type: Map<string => Array<@scope/jsii-calc-lib.Value>>
+ │   │   ├─┬ curr property (experimental)
+ │   │   │ └── type: @scope/jsii-calc-lib.Value
+ │   │   ├─┬ maxValue property (experimental)
+ │   │   │ └── type: Optional<number>
+ │   │   └─┬ unionProperty property (experimental)
+ │   │     └── type: Optional<jsii-calc.Add | jsii-calc.Multiply | jsii-calc.Power>
+ │   ├─┬ class MethodNamedProperty (experimental)
```

Comment:
Missing namespaces?

Reply:
Yah the tool doesn't show those... Hmm. I'll see what should be the best way here.